stories/dropdown/mobileExample.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import MobileDropdown from 'src/shared-components/dropdown/mobileDropdown';
+
+const DropdownContainer = styled.div`
+  padding-top: 30px;
+  max-width: 500px;
+  margin: 0 auto;
+`;
+
+export const MobileExampleDoc = `
```

Comment:
We should just move this over to the `docs/` folder and create a `docs/dropdown/` directory so we have all documentation in one place